spack/package.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Arbor(CMakePackage, CudaPackage):
+    """Arbor is a high-performance library for computational neuroscience
+    simulations."""
+
+    homepage = 'https://github.com/arbor-sim/arbor/'
+    git      = 'https://github.com/arbor-sim/arbor.git'
+    url      = 'https://arbor.readthedocs.io/en/latest/'
+    maintainers = ['bcumming', 'halfflat']
+
+    version('master', branch='master', submodules=True)
+    version('0.5', 'd0c8a4c7f97565d7c30493c66249be794d1dc424de266fc79cecbbf0e313df59', url='https://github.com/arbor-sim/arbor/releases/download/v0.5/arbor-v0.5-full.tar.gz')
```

Comment:
I find another commithash for the v0.5 tag:
[2158d2c61c0608363c66b9665ca4785e8c290b37](https://github.com/arbor-sim/arbor/releases/tag/v0.5)

Reply:
It's not the commit hash, it's the tarball hash

Reply:
I double checked, and that is the correct sha256 hash for the tar ball.